lib/modules/datasource/gitlab-tags/readme.md

```diff
@@ -32,4 +32,4 @@ Now you may use comments in your `versions.ini` files to automatically update de
 NKJS_VERSION=3.4.0
```

Comment:
We have some docs about semver-coerced in `lib/modules/versioning/semver-coerced/readme.md`. That file needs some changes:

- Follow one sentence per line pattern
- Check if the information is correct now that you've changed the default behavior

Reply:
Adapted following your input. 
One note as I'm not sure if this is clear. I haven't changed the default **of** `semver-coerced` but this changes the default versioning from `semver` to `semver-coerced`.